app/.env

```diff
@@ -4,4 +4,4 @@ MEMCACHED_PORT=11211
 
 # FICTIONAL SOCIAL API
 FICTIONAL_SOCIAL_API_HOST=https://api.supermetrics.com
-FICTIONAL_SOCIAL_API_CLIENT_ID=qwerty
+FICTIONAL_SOCIAL_API_CLIENT_ID=ju16a6m81mhid5ue1z3v2g0uh
```

Comment:
This turned out to be more interesting and I thought it would be.
Mainly because . files are excluded from search by default in VS code :DDD
So, took me a bit longer to find the right file.

Interesting puzzle though. Makes sure that candidates read description.